examples/napi/__test__/typegen.spec.ts.md

```diff
@@ -73,15 +96,30 @@ Generated by [AVA](https://avajs.dev).
     export function threadsafeFunctionThrowError(cb: (...args: any[]) => any): void␊
     export function threadsafeFunctionFatalMode(cb: (...args: any[]) => any): void␊
     export function getBuffer(): Buffer␊
+    /**␊
+     * \`constructor\` option for \`struct\` requires all fields to be public,␊
+     * otherwise tag impl fn as constructor␊
+     * #[napi(constructor)]␊
+     */␊
     export class Animal {␊
+      /** Kind of animal */␊
       readonly kind: Kind␊
+      /** This is the constructor */␊
       constructor(kind: Kind, name: string)␊
+      /** This is a factory method */␊
       static withKind(kind: Kind): Animal␊
       get name(): string␊
       set name(name: string)␊
+      /**␊
+      * This is a␊
```

Comment:
should have one more leading space

Reply:
Ah yes. I think the trailing `*/` needs an extra space too!